src/ProxyKit.Tests/PassthroughProxyTest.cs

```diff
@@ -272,11 +272,11 @@ public async Task Request_body_should_stay_as_is(string httpMethod)
 
         [Theory]
         [InlineData("GET")]
-        [InlineData("POST")]
-        [InlineData("TRACE")]
-        [InlineData("PUT")]
-        [InlineData("DELETE")]
-        [InlineData("PATCH")]
+        //[InlineData("POST")]
```

Comment:
@damianh This actually was a pretty good failing test as well.. I would argue that a post/trace/put/patch/delete should be forwarded, even if the contentlength header is not set.

Reply:
Yes the body should be forwarded for all methods, Why did you comment out these?

Reply:
Because the test checks that the body is NOT forwarded for these methods. So actually it showed the issue with the implementation where the testserver wasn't forwarding the content length.

Reply:
Hmm. I think the body needs to be forwarded if it's there regardless of the HTTP Method. The previous issue was it was sending a 0 byte body with content-length of 0 when the incoming request had no content-length at all.